app/services/pii/store.rb

```diff
@@ -0,0 +1,40 @@
+module Pii
+  class Store
```

Comment:
This class seems, conceptually at least, to overlap significantly with `Pii::Cacher` -- the chief difference seems to be that `Pii::Store` "knows" much more about the (undocumented) inner details of the redis and session innards.

Maybe `Pii::SessionStore` would be clearer than the class is dealing specifically with the session innards?